src/mmrelay/matrix/sync_bootstrap.py

```
@@ -0,0 +1,1201 @@
+        else:
+            facade.bot_user_name = bot_user_id or ""  # type: ignore[assignment]
+    except NIO_COMM_EXCEPTIONS as e:
+        facade.logger.debug(f"Failed to get bot display name for {bot_user_id}: {e}")
+        facade.bot_user_name = bot_user_id or ""  # type: ignore[assignment]
+
+    cast(Any, client).e2ee_enabled = e2ee_enabled
+
+
+async def connect_matrix(
+    passed_config: dict[str, Any] | None = None,
+) -> "facade.AsyncClient | None":
```

Ruff (0.15.9)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)